src/lib/LR1121Driver/LR1121.cpp

```diff
@@ -406,8 +406,11 @@ void LR1121Driver::ConfigModParamsLoRa(uint8_t bw, uint8_t sf, uint8_t cr, SX12X
     if (radioNumber & SX12XX_Radio_1 && radio1isSubGHz)
         CorrectRegisterForSF6(sf, SX12XX_Radio_1);
 
-    if (radioNumber & SX12XX_Radio_2 && radio2isSubGHz)
-        CorrectRegisterForSF6(sf, SX12XX_Radio_2);
+    if (GPIO_PIN_NSS_2 != UNDEF_PIN)
+    {
+        if (radioNumber & SX12XX_Radio_2 && radio2isSubGHz)
+            CorrectRegisterForSF6(sf, SX12XX_Radio_2);
```

Comment:
(nitpick) Some curly braces would be tops :)

Reply:
They would be.